options/index.js

```diff
@@ -25,52 +26,56 @@ function saveOptions(e) {
     });
 
   setTimeout(() => {
-    saveStatus.textContent = ''
-  }, 3000)
+    saveStatus.textContent = 'Save all'
+  }, 1500)
```

Comment:
That's too short for me
```suggestion
  }, 3000)
```